src/resources.js

```diff
@@ -813,9 +913,15 @@ export class Resources {
     }
     if (request) {
       request.newHeight = newHeight;
+      request.force = force || request.force;
+      request.fallback = fallback || request.fallback;
     } else {
-      this.changeHeightRequests_.push(
-          {resource: resource, newHeight: newHeight});
+      this.changeHeightRequests_.push({
+        resource: resource,
+        newHeight: newHeight,
+        force: force,
+        fallback: fallback
+      });
```

Comment:
probably not a part of our style guide but a nice to know is ES2015 Object Literal Property Value Shorthand. (when key and the value have the same identifier)

```
this.changeHeightRequests_.push({
        resource,
        newHeight,
        force,
        fallback
      });
```